website/docs/basics/blocks-intro.md

```diff
@@ -29,16 +31,26 @@ Yes, we do. All block variations are located in the `src/Blocks/variations` fold
 
 ### What is a wrapper?
 
+All details about wrapper you can find in [this chapter](blocks-wrapper).
+
 ### Do you support inner blocks?
 
+Yes, we support everything that core natively supports. How to use them you can find in [this chapter](block-manifest).
+
 ### Why do you use manifest.json in all blocks and components?
 
+So we can provide content, attributes, options and much more across multiple different files in multiple programing languages. More details read in [this chapter](block-manifest).
```

Comment:
```suggestion
So that we can provide content, attributes, options, and much more across multiple different files in multiple contexts (in both PHP and JS). You can read all about it in [this chapter](block-manifest).
```